flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/FileCatalogStoreFactory.java

```diff
@@ -20,42 +20,39 @@
 
 import org.apache.flink.configuration.ConfigOption;
 import org.apache.flink.configuration.ReadableConfig;
-import org.apache.flink.table.catalog.exceptions.CatalogException;
 import org.apache.flink.table.factories.CatalogStoreFactory;
-import org.apache.flink.table.factories.FactoryUtil;
+import org.apache.flink.table.factories.FactoryUtil.FactoryHelper;
 
+import java.util.Collections;
 import java.util.HashSet;
 import java.util.Set;
 
-import static org.apache.flink.table.catalog.FileCatalogStoreFactoryOptions.CHARSET;
 import static org.apache.flink.table.catalog.FileCatalogStoreFactoryOptions.IDENTIFIER;
 import static org.apache.flink.table.catalog.FileCatalogStoreFactoryOptions.PATH;
 import static org.apache.flink.table.factories.FactoryUtil.createCatalogStoreFactoryHelper;
 
-/** CatalogStore factory for {@link FileCatalogStore}. */
+/** Catalog store factory for {@link FileCatalogStore}. */
 public class FileCatalogStoreFactory implements CatalogStoreFactory {
 
     private String path;
 
-    private String charset;
-
     @Override
     public CatalogStore createCatalogStore() {
-        return new FileCatalogStore(path, charset);
+        return new FileCatalogStore(path);
     }
 
     @Override
-    public void open(Context context) throws CatalogException {
-        FactoryUtil.FactoryHelper factoryHelper = createCatalogStoreFactoryHelper(this, context);
+    public void open(Context context) {
+        FactoryHelper<CatalogStoreFactory> factoryHelper =
+                createCatalogStoreFactoryHelper(this, context);
         factoryHelper.validate();
-        ReadableConfig options = factoryHelper.getOptions();
 
+        ReadableConfig options = factoryHelper.getOptions();
         path = options.get(PATH);
-        charset = options.get(CHARSET);
     }
 
     @Override
-    public void close() throws CatalogException {}
+    public void close() {}
```

Comment:
Call super.close()

Reply:
There are no super implementation here, this class implements the interface directly.